tests/waku_filter_v2/test_waku_client.nim

```diff
@@ -18,6 +18,7 @@ import
   ../testlib/wakucore,
   ../testlib/testasync,
   ../testlib/testutils,
+  ../testlib/futures,
```

Comment:
Why is a new import required if no code is being added to this file?

Reply:
Because I moved the `newPushHandlerFuture` in `waku_filter_utils` (already imported) into the new `futures` module, as that function will also be used outside waku filter tests.